llvm/lib/Target/DirectX/DXILShaderFlags.h

```diff
@@ -91,7 +91,10 @@ struct ModuleShaderFlags {
   const ComputedShaderFlags &getCombinedFlags() const { return CombinedSFMask; }
 
 private:
-  bool CanSetResMayNotAlias;
+  // Booleans set by module flags
+  bool CanSetResMayNotAlias;   // dx.resmayalias
+  bool NativeLowPrecisionMode; // dx.nativelowprec
```

Comment:
Should we name these similarly to reflect their similar usage/purpose? Does `CanSetNativeLowPrecision` make sense? If that feels forced I think `HasResMayAlias` and `HasNativeLowPrec` wouldn't be bad, though that would require flipping the value of the existing member.

Reply:
I think `CanSetNativeLowPrecisionMode` would make more sense